### DISCUSSION TITLE
Configuration for oauth authorization with spring kafka

### DISCUSSION THREAD

**Discussion by ramgopalcheedu** [spring-projects/spring-kafka#1796] [Q&A]
Hello,

We are planning to use Spring kafka template for producing messages. Our kafka setup only takes oauth tokens for authorization. We have custom token generation and we are interested in Spring kafka properties that support oauth

**Comment by garyrussell** [comment]
This is a question for the Kafka folks; Spring delegates all connection activity to the `kafka-clients` - try one of their mailing lists.

https://kafka.apache.org/contact
